spec.html

```diff
@@ -38322,23 +38322,17 @@ <h1>`Promise.any` Reject Element Functions</h1>
         </emu-clause>
       </emu-clause>
 
-      <emu-clause id="sec-promise.prototype">
-        <h1>Promise.prototype</h1>
-        <p>The initial value of `Promise.prototype` is the Promise prototype object.</p>
-        <p>This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *false* }.</p>
-      </emu-clause>
-
```

Comment:
this should probably be restored

Reply:
It is still in there, just pushed to bottom because of alphabetical order sorting